cmd/minikube/cmd/start.go

```diff
@@ -305,6 +306,10 @@ func startWithDriver(starter node.Starter, existing *config.ClusterConfig) (*kub
 	return kubeconfig, nil
 }
 
+func warnAboutMultiNode() {
+	out.T(out.Tip, "Multi-node clusters are currently experimental and might exhibit unintended behavior.\nTo track progress on multi-node clusters, see https://github.com/kubernetes/minikube/issues/7538.")
```

Comment:
out.WarningT?

Reply:
I wasn't sure if I wanted to pollute std err with this message, but it makes sense. Fixed.